src/ToricVarieties/NormalToricVarieties/constructors.jl

```diff
@@ -510,7 +576,7 @@ function Base.show(io::IO, v::AbstractNormalToricVariety)
     end
 
     properties_string = [join(properties_string, ", ")]
-    push!(properties_string, "toric variety")
+    push!(properties_string, "toric variety over " * lowercase(string(coefficient_ring(v))))
```

Comment:
As we discussed earlier this week, I have adjusted the print method such that for `QQ` and `ZZ` as coefficient ring, we print `QQ` and `ZZ`, respectively. In the other cases it prints `coefficient_ring(variety)`. Unfortunately, `string(variety)` returns the value of `Base.show` rather than the name of the variable which in the current session encodes the toric variety in question. But maybe this is acceptable?